### ISSUE TITLE
ResponseCurve for multiple covariates

### ISSUE THREAD

**Issue by timcdlucas** [zoonproject/zoon#170]
I'm not sure what the best solution is, but ResponseCurve currently defaults to printing each map in turn, too quickly to be seen.

Perhaps an option to save all the figures somewhere. Or a pause until click like `plot.lm` has. Even a 3 second pause between plots might help.

**Comment by goldingn** [comment]
On RStudio you can flick back through them...

I think Greg wants to do a multi-predictor response-curve plot, in which case, we'd just set this to do a close-up on a single response of interest.

Interactive HTML graphics would be a nice addition, allowing users to flick between them, had a look previously but didn't get very far I'm afraid

**Comment by timcdlucas** [comment]
Gah, RStudio. I think the workshop validated my dislike.

I guess it should at least save the plot to the workflow object? Then you can 

    print(m7$report[[1]][[1]])

or whatever. I guess this applies to `PrintMap` as well.

**Comment by goldingn** [comment]
Yeah, something like that (but with fewer square brackets :)).

Executing modules in child processes would mean no side-effects like plotting to the console. Therefore everything would have to be saved, and the workflow print method could re-plot all of the saved images and html widgets. Perhaps in a report via links / embedding?

We could make that easy for module developers by setting up graphics devices for each output module in workflow. That would still work with child processes.

**Comment by AugustT** [comment]
I think it would be good to have plots returned to console, this provides instant feedback and is nice for the users. I guess this could be an HTML report instead, but the key thing in my mind is to make the results visible (ie not having to hunt around in the workflow object) and to make it transparent to the lay audience, how to save the plots they get.

**Comment by timcdlucas** [comment]
Yes I agree. It's one of the main ways I test if something is working. Run a workflow and check the image that gets printed (usually a map).

So I think console + perhaps in the object as well is a good way.

**Comment by goldingn** [comment]
why not just return the report as a side effect of workflow? That would give instant feedback, but all the imnformation.

**Comment by AugustT** [comment]
Like I say I think the HTML report is fine as long as the results are visible and saving results is easy. It would be nice if we could get the report to display in the RStudio viewer as that might make it more 'familiar' to lay users.
